curvefs/src/client/common/config.cpp

```diff
@@ -196,12 +196,12 @@ void InitVolumeOption(Configuration *conf, VolumeOption *volumeOpt) {
                               &volumeOpt->allocatorOption.type);
 
     conf->GetValueFatalIfFail(
-        "volume.blockgroup.allocate_once",
+        "volume.blockGroup.allocate_once",
         &volumeOpt->allocatorOption.blockGroupOption.allocateOnce);
 
     if (volumeOpt->allocatorOption.type == "bitmap") {
         conf->GetValueFatalIfFail(
-            "volume.bitmapallocator.size_per_bit",
+            "volume.bitmapAllocator.size_per_bit",
             &volumeOpt->allocatorOption.bitmapAllocatorOption.sizePerBit);
         conf->GetValueFatalIfFail(
             "volume.bitmapallocator.small_alloc_proportion",
```

Comment:
ditto

Reply:
conf->GetValueFatalIfFail(
            "volume.bitmapallocator.small_alloc_proportion",

Reply:
im sry for missing one unchanged

Reply:
> conf->GetValueFatalIfFail( "volume.bitmapallocator.small_alloc_proportion",

done

Reply:
good job, lets us wait the ci finished

Reply:
> CI still running, is there some failure occured?

Reply:
> > CI still running, is there some failure occured?

http://59.111.91.248:8080/job/opencurve_multijob/6935/ 

as you can see,  there is running the failover test.

Reply:
the failover test log shows:
```
------------------------------------------------------------------------------
test loop mount umount                                                | FAIL |
mount/umount error
------------------------------------------------------------------------------
```